src/Controller/ResourcesController.php

```diff
@@ -50,25 +36,19 @@ public function resourcesAction(Request $request) : Response
             ArticleSection::basic('Images', 2, $this->render(
                 new Paragraph('A picture is worth a thousand words and we use imagery to create impact and capture the imagination of our readers. In our Flickr channel you’ll find images carefully selected from research articles or specifically created, available to use under the CC BY 2.0 licensing agreement.'),
                 Listing::unordered([
-                    '<a href="https://www.flickr.com/photos/128643624@N07/">Visit Flickr channel</a>',
+                    '<a href="https://www.flickr.com/photos/109374423@N04/albums">Visit Flickr channel</a>',
```

Comment:
This could be extracted and go up now?

Reply:
This will be going up Monday so do not think a couple of days will make a difference.